Examples/Algorithms/Fitting/src/FittingAlgorithm.cpp

```diff
@@ -113,13 +114,14 @@ ActsExamples::ProcessCode ActsExamples::FittingAlgorithm::execute(
       } else {
         ACTS_DEBUG("No fitted paramemeters for track " << itrack);
       }
-      // Create a SimMultiTrajectory
+      // store the result
       trajectories.emplace_back(std::move(fitOutput.fittedStates),
                                 std::move(trackTips), std::move(indexedParams));
     } else {
       ACTS_WARNING("Fit failed for track " << itrack << " with error"
                                            << result.error());
-      // Fit failed, but still create an empty SimMultiTrajectory
+      // Fit failed. Add an empty result so the output container has
+      // the same number of entries as the input.
```

Comment:
Arguably better comment, but unrelated to the changeset

Reply:
As above.